src/main/java/shapes/Demo.java

```diff
@@ -13,10 +15,14 @@ public static void main(String[] args) {
                 System.out.println("Area of circle = " + ((Circle) shape[i]).getArea());
             }
             if (shape[i] instanceof Rectangle) {
-                System.out.println("Perimeter of rectangle = "+((Rectangle) shape[i]).getPerimeter());
+                System.out.println("Perimeter of rectangle = " + ((Rectangle) shape[i]).getPerimeter());
             }
+            if (shape[i].isInside(x, y)==true) {
```

Comment:
no need to `== true` since if block will be executed if `shape[i].isInside(x,y)`